addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -43,7 +47,11 @@
 "300050","300050","Capital","equity","False"
 "300060","300060","Dividends","equity","False"
 "300070","300070","Drawings","equity","False"
+"360000","360000","Profit and Loss (This Year)","equity_unaffected","False"
 "400010","400010","Undistributed Profits/Losses","equity_unaffected","False"
+"422003","422003","Discount Received","income","False"
+"424002","424002","Forex Exchange Gain or Loss","income","False"
```

Comment:
See 610440, should we split the gains and losses accounts?
"Foreign Exchange Gain or Loss" - I think this is a duplicate, it could be an expense or income,
so it's probably best to split into 
"Foreign Exchange Gains", Income
"Foreign Exchange Expenses", Expenses

Reply:
I actually raised this question to the partner and he said that they usually use the income account that they debit in case of income and credit in case of expense. But I agree with you that it is cleaner from an accounting perpsective to split the accounts. Let's do that.